CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## 38 Release Notes
 
+### HPC paths with Docker
+
+The `ConfigBackendLifecycleActorFactory` path variables `script`, `out` and `err` are now consistent when running with
+and without docker.
+
+Without Docker:
+- `script` - The path of the script to be run.
+- `out` - The path to the stdout.
+- `err` - The path to the stderr.
+
+With Docker:
+- `script` - The path outside the container of the script to be run.
+- `out` - The path outside the container to the stdout.
+- `err` - The path outside the container to the stderr.
+- `docker_script` - The path within the docker container of the `script`.
+- `docker_out` - The path within the docker container of the `out`.
+- `docker_err` - The path within the docker container of the `err`.
```

Comment:
Personal opinion: this feels *looooooong* for a changelog entry... Is there any way of linking out to an appropriate readthedocs page?

Reply:
Will put a "see the docs..." style link